src/socks-to-rtc/socks-to-rtc.ts

```diff
@@ -163,7 +163,10 @@ module SocksToRtc {
       this.onceStopped_ = this.onceStopping_.then(this.stopResources_);
       this.onceStopped_.then(this.dispatchEvent_.bind(this, 'stopped'));
 
-      return onceReady;
+      var rejectOnStopping = new Promise((F, R) => {
```

Comment:
Freedom doesn't support fields, so it would have to be a Promise-getter method or a 'started' event.  Anyway, nothing was using it.  Currently we only really have one start method (`startInternal` is just for testing) so I think it's OK.